wsidicomizer/encoding.py

```diff
@@ -116,7 +121,7 @@ def __init__(
 
         """
         self._quality = quality
-        if self.quality < 1:
+        if self._quality < 1:
```

Comment:
As self._quality is set the more descriptive quality-property should be OK to use?

Reply:
I am used to avoiding making method calls in constructors from other languages and thus accessed the member variable directly. I guess this is stylistic and can be ignored.